docs/source/ko/_toctree.yml

```
@@ -172,8 +172,8 @@
       title: 다중 GPU에서 훈련 진행하기
     - local: in_translation
       title: (번역중) Fully Sharded Data Parallel
-    - local: in_translation
-      title: (번역중) DeepSpeed
+    - local: DeepSpeed
```

This should just be the file name. You can also remove the other instance of `deepspeed` in the toctree below

```suggestion
    - local: deepspeed
```